### PR TITLE
0055 MetaMask Snap Proposal

### PR DESCRIPTION
**Pull request by JST5000**
| [View Proposal Here](https://github.com/XRPLBounties/Proposals/pull/55/files?short_path=8dc7af2#diff-8dc7af291634aeeb6e25325e62d62c1d5a916c823c46522645ac334105e79236) |
| - |

(This is a re-upload of this PR since it was merged by accident, then reverted)

### CODE REVIEW

**Review by pdp2121 — COMMENTED** [review]
Are we still putting this on hold for now?

**Comment by JST5000** [comment]
> Are we still putting this on hold for now?

Yeah we still need to have discussions around budget approval - Needed to recreate this PR though in order to track our decision on that front and correctly document the status of this bounty in the Discussion post for it here: https://github.com/XRPLBounties/Proposals/discussions/42

**Review by ckniffen — APPROVED** [review]

**Review by khancode — APPROVED** [review]

**Comment by intelliot** [comment]
The team has decided to pause on this bounty for now, so it will not be available for the time being.

**Pull request closed without merging**
